guides/deploy/starter_deploy_kubernetes.html

```diff
@@ -0,0 +1,83 @@
+<html lang="en">
+
+<head>
+  <title>Learn - Deploy using Kubernetes and Docker</title>
+  <link rel="icon" type="image/png" href="../../assets/favicon-32x32.png" sizes="32x32" />
+  <link rel="icon" type="image/png" href="../../assets/favicon-16x16.png" sizes="16x16" />
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="../../scripts/prism.js"></script>
+  <link rel="stylesheet" href="../../css/reset.css">
+  <link rel="stylesheet" href="../../css/main.css">
+  <link rel="stylesheet" href="../../css/guides.css">
+  <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/mobile_guides.css">
+</head>
+
+<header>
+  <div class="header-container">
+    <div class="header-main">
+      <a class="home-link" href="../../index.html">
+        <img class="header-logo" src="../../assets/kitura-logo.png" alt="Kitura logo">
+        <h1 class="header-title">KITURA</h1>
+      </a>
+    </div>
+    <nav class="header-nav">
+      <a class="header-link active-nav" href="../../learn.html">LEARN</a>
+      <a class="header-link" href="../../packages.html">PACKAGES</a>
+      <a class="header-link" href="../../events.html">EVENTS</a>
+      <a class="header-link" href="../../help.html">HELP</a>
+    </nav>
+  </div>
+</header>
+<body>
+  <section class="guide-content">
+    <div class="title-block">
+        <img width="480px" src="../../assets/Kitura.svg" alt="Kitura Logo">
+      <h1>DEPLOY YOUR APP USING KUBERNETES AND DOCKER</h1>
+    </div>
+
+<p>This guide will show you how to deploy the Kitura application you created by following our <a href="https://www.kitura.io/guides/gettingstarted">Getting Started</a> guide to a <a href="https://kubernetes.io/">Kubernetes</a> based environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud and <a href="https://console.bluemix.net/docs/containers/container_index.html#container_index">create a Kubernetes cluster</a> for free, or <a href="https://kubernetes.io/docs/setup/minikube/">set one up locally using minikube</a>.</p>
+<h4 id="starter-project-contents">Starter project contents</h4>
+<p>Your generated Kitura project already contains Docker related files and a <a href="https://docs.helm.sh/developing_charts/">Helm chart</a> used to describe the set of Kubernetes resources it requires.</p>
+<p>To deploy your application to Kubernetes, we first need to build it into a Docker container for Kubernetes to execute. Your project includes the following files for Docker support: - .dockerignore - Dockerfile - Dockerfile-tools</p>
```

Comment:
: - .dockerignore - Dockerfile - Dockerfile-tools
this format isn't readable when rendered. Perhaps a list would be better?